### DISCUSSION TITLE
22.0.1

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1478] [General]
## Releases Notes for 22.0.1
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.1/draw.io-22.0.1-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.1/draw.io-22.0.1-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.1/draw.io-universal-22.0.1.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.1/drawio-amd64-22.0.1.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.1/drawio-x86_64-22.0.1.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.1/drawio-x86_64-22.0.1.rpm)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates to [draw.io core 22.0.1](https://github.com/jgraph/drawio/blob/v22.0.1/ChangeLog). All changes from 22.0.1 are added in this build.
- Attempted fix for #1470 

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v22.0.1'>22.0.1</a>.</em>

**Comment by tienlx91** [comment]
after update to version 22.0.1, the copy/paste feature no longer works, I have to revert back to 22.0.0

**Comment by vadimplSPb** [comment]
Forced to use Ctrl+drag instead of Copy/Paste
